database/seeds/RegistrationsSeeder.php

```diff
@@ -37,9 +37,120 @@ public function run()
         // create a registration with a bundle
         $bundle = factory(App\Bundle::class)->create();
         factory(App\Registration::class)->create()->bundles()->save($bundle);
-    }
+
+        // centre without SK rules for fam1 & fam4 that is not in same area as families 3, 5 and 6
+        $testCentreFam1And4 = App\Centre::find(7);
+        $testCentreFam1And4->sponsor_id = App\Sponsor::find(5)->id;
+
+        /*
+        centre with SK rules for fam2 that is not in same area as families 3, 5 and 6
+        TODO: add SK rules
+        */
+        $testCentreFam2 = App\Centre::find(2);
+        $testCentreFam2->sponsor_id = App\Sponsor::find(2)->id;
+
+        /*
+        centre with SK rules for fam3 that is in a different area from all other families
+        TODO: check and/or add SK rules
+        */
+        $testCentreFam3 = App\Centre::find(1);
+        $testCentreFam3->sponsor_id = App\Sponsor::find(1)->id;
+
+        /*
+        centre with SK rules but without toggle for fam 5 that is in a different area from all other families
+        TODO: check and/or add SK rules without toggle
+        */
+        $testCentreFam5 = App\Centre::find(3);
+        $testCentreFam5->sponsor_id = App\Sponsor::find(3)->id;
+
+        /*
+        centre with non SK rules for fam 6 that is in a different area from all other families
+        TODO: check that we don't have SK rules
+        */
+        $testCentreFam6 = App\Centre::find(6);
+        $testCentreFam6->sponsor_id = App\Sponsor::find(4)->id;
+
+        $familyData1 = [
```

Comment:
It might be clearer to have the `$testCentreFam1` passed in this array also, like

`
```
'center' => $testCentreFam1`,
'carers' => [[...]],
...
```